### DISCUSSION TITLE
The compatibility issue between the matplotlib and python.

### DISCUSSION THREAD

**Discussion by ZigongXu** [jgieseler/Solar-MACH#23] [Issues]
The current matplotlib  3.5.2 is not compatible with Python 3.11. This will cause an error while installing the requirement.txt:
'''
Failed to build matplotlib ERROR: Could not build wheels for matplotlib, which is required to install pyproject.toml-based projects
'''
Should make it clear which python version should be used

**Comment by jgieseler** [comment]
Yes, the documentation of https://github.com/jgieseler/Solar-MACH is quite rudimentary due to lack of time. This is also because I think most users either go directly to the website https://solar-mach.github.io or they use the (at https://github.com/jgieseler/Solar-MACH mentioned) [PyPI package](https://pypi.org/project/solarmach/), which indeed has a full documentation.

I added now the Python version badge [![Python](https://img.shields.io/pypi/pyversions/solarmach?style=flat&logo=python)](https://pypi.org/project/solarmach/) from the PyPI package, so that it will automatically reflect the latest status.
